users.tf

```diff
@@ -54,6 +54,8 @@ locals {
         "devops"
       ],
       eks_access = [
+        local.eks_cluster_testnet,
+        local.eks_cluster_mainnet,
```

Comment:
@ArseniiPetrovich I'm not sure we need to grant administrative permissions to our developers.
maybe we can grant them a limited list of permissions?